girder/girder_large_image/web_client/views/imageViewerWidget/geojs.js

```diff
@@ -67,43 +67,20 @@ var GeojsImageViewerWidget = ImageViewerWidget.extend({
 
         var geo = window.geo; // this makes the style checker happy
 
-        var params;
-        if (!this.metadata.geospatial || !this.metadata.bounds) {
-            var w = this.sizeX, h = this.sizeY;
-            params = geo.util.pixelCoordinateParams(
-                this.el, w, h, this.tileWidth, this.tileHeight);
-            params.layer.useCredentials = true;
-            params.layer.url = this._getTileUrl('{z}', '{x}', '{y}');
-            if (this.tileWidth > 8192 || this.tileHeight > 8192) {
-                params.layer.renderer = 'canvas';
-            }
-            this.viewer = geo.map(params.map);
-            params.layer.autoshareRenderer = false;
-            this._layer = this.viewer.createLayer('osm', params.layer);
-            if (this.metadata.frames && this.metadata.frames.length > 1) {
-                setFrameQuad(this.metadata, this._layer, {
-                    // allow more and larger textures is slower, balancing
-                    // performance and appearance
-                    // maxTextures: 16,
-                    // maxTotalTexturePixels: 256 * 1024 * 1024,
-                    baseUrl: this._getTileUrl('{z}', '{x}', '{y}').split('/tiles/')[0] + '/tiles',
-                    restRequest: restRequest,
-                    restUrl: 'item/' + this.itemId + '/tiles',
-                    query: 'cache=true'
-                });
-                this._layer.setFrameQuad(0);
-            }
-        } else {
-            params = {
-                keepLower: false,
-                attribution: null,
-                url: this._getTileUrl('{z}', '{x}', '{y}', {encoding: 'PNG', projection: 'EPSG:3857'}),
-                useCredentials: true,
-                maxLevel: this.levels - 1
-            };
-            // the metadata levels is the count including level 0, so use one
-            // less than the value specified
+        var params = {
+            useCredentials: true,
+            autoshareRenderer: false,
+            keepLower: false,
```

Comment:
I had included this because it was in the `params` object on line 98 previously, but now I understand that this was only intended for the geospatial cases. I can move this to the geospatial `if` block instead.